RELEASE/scripts/autoscend/combat/auto_combat_default_stage5.ash

```diff
@@ -31,6 +31,16 @@ string auto_combatDefaultStage5(int round, monster enemy, string text)
 	retval = auto_combatFallOfTheDinosaursStage5(round, enemy, text);
 	if(retval != "") return retval;
 
+	//with loofah, you can stagger and deal cold or hot damage
+	if(canUse($skill[loofah stew]) && monster_element(enemy) != $element[cold])
```

Comment:
Is there risk of these not doing enough damage? At low levels, this is probs good, but concerned about higher levels. Wiki doesn't seem to have damage range well spaded. Do you have a good idea of how much damage stew and lava do?

Ideally we would cast `if(can do damage based on enemy element && (canStagger || killEnemyWithNextHit))`

This way we don't cast against big enemies that will squish us. If ya want to get extra fancy, `killEnemyWithNextHit` should account for elemental weakness. 

I could have a similar comment for stuff added to stage 4 but those give a benefit. Whereas stew and lava are only damage, perhaps we want to use a bigger damage spell at this point if the enemy is big.

Kinda feel like I'm over thinking this... lmk if that is the case haha

Reply:
If they always stagger, does it matter?  Anyway, when wearing the full gear, I've seen lava and stew each do, I want to say, around 250, with less gear worn causing less damage.  That would be interesting to spade.

Reply:
If they always stagger then no, but I believe all stager sources fail at >= 150 bonus ML based on the wiki

https://kol.coldfront.net/thekolwiki/index.php/Monster_Level
![image](https://github.com/Loathing-Associates-Scripting-Society/autoscend/assets/4781473/9b86ff4d-38b4-4da0-9991-9b0aa78bda6b)

150 is a whole bunch of ML but it can happen